src/canvas/canvas.interface.ts

```diff
@@ -68,12 +70,16 @@ export class CanvasMetadataUpdateDTO {
   @MinLength(3)
   @MaxLength(255)
   name: string;
+  @MaxLength(4000)
+  description: string;
 }
 
 export class CanvasCreateDTO {
   @MinLength(3)
   @MaxLength(255)
   name: string;
+  @MaxLength(4000)
+  description: string;
```

Comment:
description should be optional field, currently we will get http 400 on create and update endpoints

Reply:
@coredumped7893 Done